posit-dev/positron
# python: detect app framework

- creates context key for is/is not app
- "Run App in Terminal" will show up when in an app

We might want to think about how this interacts with the Shiny extension. I guess at this point, it will just populate a LOT of options in the Play button, which is ugly but benign.

### QA Notes




create a `.py` file

```
import streamlit as st
x = st.slider('x')  # 👈 this is a widget
st.write(x, 'squared is', x * x)
```

open and click on play button, you should see 

![Screenshot 2024-09-12 at 4 01 23 PM](https://github.com/user-attachments/assets/afacee31-e6cc-48d8-b185-1adcfd3bf95d)

(clicking on this file will only save the file and have no other affect)
